### PR TITLE
Add RDN Token support

### PR DESCRIPTION
**Pull request by ice09**
##### Description

Add RDN Token support for gitcoin
RDN Token: https://etherscan.io/token/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6
Raiden Info: https://raiden.network/ 

##### Checklist

- [X] linter status: 100% pass
- [X] changes don't break existing behavior
- [ ] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

* dashboard
* crypto

##### Refers/Fixes
Refs: #155

### CODE REVIEW

File: app/dashboard/tokens.py [hunk 1]
```diff
@@ -21,8 +21,6 @@
     {"addr": "0x2941deaad71adb02b944bd38ebce2f1f4c9a62dc", "name": "COLO", "decimals": 18},
     # from https://github.com/etherdelta/etherdelta.github.io/blob/master/config/main.json
     {"addr": "0x0000000000000000000000000000000000000000", "name": "ETH", "decimals": 18},
-    # for backwards compat with old data, also accept 0x0 as ETH token address
-    {"addr": "0x0",                                        "name": "ETH", "decimals": 18},
```
Comment (owocki) [inline app/dashboard/tokens.py]: any reason you removed this line?

**Comment by owocki** [comment]
left 1 comment.  otherwise LGTM

File: app/dashboard/tokens.py (hunk 1 above)
Comment (ice09) [reply app/dashboard/tokens.py]: Sorry for this, no that was a mistake. Should be fine now.

**Comment by owocki** [comment]
thanks :)

**Pull request merged**